cabal.ghc948.project

```diff
@@ -12,6 +12,10 @@ else
 constraints: http-io-streams -brotli,
              any.aeson >= 2.0.1.0
 
+if os(mingw32)
+  if impl(ghc >= 9.4)
+    constraints: language-c >= 0.9.3
+
```

Comment:
where did this come from?

Reply:
Those changes come from the re-freeze, which were not consistently done after every single change to the project files.